data/human/outfits.txt

```diff
@@ -394,6 +394,14 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "The old mans sword"
+	category "Hand to Hand"
+	cost 120000
+	thumbnail "outfit/old sword"
+	"capture attack" 3
+	"capture defense" 3
+	"unplunderable" 1
+	description "A sword that has all kind of alien technology implemented in it. You have yet to find out what it can do. Some functionalities are a personall shield and an implemented shooting mechanism."
```

Comment:
```suggestion
outfit "Old Man's Sword"
	category "Hand to Hand"
	cost 120000
	thumbnail "outfit/old sword"
	"capture attack" 3
	"capture defense" 3
	"unplunderable" 1
	description "While swords are seen today as old curios, commemorative objects, or are under archaeological study, this sword seems to be as combat effective as any laser rifle. Swinging the sword discharges an energy bolt that can be used at range to hit an enemy, or neutralize one of their shots coming at you."
```
This is just an idea of what it could look like. Perhaps the description might still be improved.

Reply:
yeah that could work, so long as its not a personal shield or a gun-sword per say like we intend for the warlord blades it's good for me
I mean even if it is I'm not going to hard block the PR or smth its just that it'd be good to have varied things and if they're the same that'd limit the power of the UHai blade later on

Reply:
Here's a better one:
```suggestion
outfit "Old Man's Sword"
	category "Hand to Hand"
	cost 120000
	thumbnail "outfit/old sword"
	"capture attack" 3
	"capture defense" 3
	"unplunderable" 1
	description "Got this from a pair of siblings from New China, fulfilling their eccentric father's last will and testament. While swords are seen today as old curios, commemorative objects, or are under archaeological study, this sword seems to be as combat effective as any laser rifle, and unlike any you're seen before. Swinging the sword discharges an energy bolt that can be used at range to hit an enemy, or neutralize one of their shots coming at you."
```

Reply:
I still want to have the part of that you do not really know what this sword can also do in the description. Something like at some occasions the sword showed other functionalities that you were not able to reproduce so far.